helm/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kiam
-version: 5.5.0
+version: 5.5.1
```

Comment:
Hey @caiohasouza, thanks for contributing! 

My only suggestion is that please can we bump the minor version here, rather than patch, as a backwards-compatible functionality change is being made

Reply:
Hello @rhysemmas , right, and i need change something in my commits? Or this change will be done for you?

Reply:
You'll need to update your commit so that the minor chart version is bumped instead (`5.6.0` instead of `5.5.1`), as when this PR is merged our CI will package a new chart using the version number included in the chart.

Reply:
Ok, done, could you check please?